src/components/Dropdown/DropdownError.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { StatusAlertSmall } from 'wix-ui-icons-common/dist/src';
```

Comment:
I almost missed this.
We need to use our own icons.
You can use the icon we're using in the `TextField` and `TextArea` components.